metadata.mk

```diff
@@ -60,3 +60,5 @@ LIBBPF_VERSION=v1.4.6
 # The operator branch corresponding to this branch.
 OPERATOR_BRANCH=master
 
+# The image to use for building calico/base-dependent modules (e.g. apiserver, typha)
+CALICO_BASE=calico/base
```

Comment:
We normally define images in `lib.Makefile`. for example, the [GO_BUILD_IMAGE](https://github.com/projectcalico/calico/blob/master/lib.Makefile#L138) definition. Please move this closer to where it is used (like [here](https://github.com/projectcalico/calico/blob/master/lib.Makefile#L264)) and also use the `?=` operator so that we can parameterize it in the command line.